app/assets/stylesheets/common/admin/admin_base.scss

```
@@ -284,6 +289,14 @@ $mobile-breakpoint: 700px;
     .go-back {
       margin-top: 1em;
     }
+    .desc {
+      padding-top: 3px;
+      font-size: var(--font-down-1);
+      line-height: var(--line-height-large);
+    }
+    .desc {
+      color: var(--primary-medium);
+    }
```

you can mix these two lines:

```suggestion
    .desc {
      padding-top: 3px;
      font-size: var(--font-down-1);
      line-height: var(--line-height-large);
      color: var(--primary-medium);
    }
```